.gitignore

```diff
@@ -33,6 +33,7 @@ website/build/bloop-gh-pages/
 node_modules/
 package-lock.json
 .metals/
+project/metals.sbt
```

Comment:
Hmmm, Bloop loads the sbt-bloop plugin itself. You should not have this directory in your setup, otherwise bad things can happen (TM). The `-Dbloop.generate.sbt=false` option in `.jvmopts` is there so that Metals doesn't write this file and instead blindly assumes the plugin is already added in the project. Is that not being picked up somehow in your computer?

Reply:
Och, that must have been an old file, sorry about that.

Reply:
For the record: removing `project/metals.sbt` actually resolved the issue for me!